aws/rds-postgres/db.tf

```diff
@@ -321,21 +319,24 @@ data "aws_kms_key" "parameter_store_key" {
 }
 
 module "rds_ssm_param_secret" {
+  enabled        = var.password == "" ? "true" : "false"
   source         = "git::https://github.com/cloudposse/terraform-aws-ssm-parameter-store?ref=0.1.5"
   kms_arn        = "alias/parameter_store_key"
-  parameter_read = ["/${local.module_prefix}/rds-secret"]
+  parameter_read = ["/${local.stage_prefix}/${var.name}-password"]
 }
 
 module "rds_ssm_param_username" {
+  enabled        = var.username == "" ? "true" : "false"
   source         = "git::https://github.com/cloudposse/terraform-aws-ssm-parameter-store?ref=0.1.5"
-  parameter_read = ["/${local.module_prefix}/rds-username"]
+  parameter_read = ["/${local.stage_prefix}/${var.name}-username"]
```

Comment:
local.module_prefix should work here

Reply:
Unable to use module prefix due to the need to add the separating "/" as name is not part of the parameter service.